sycl/include/sycl/access/access.hpp

```diff
@@ -356,6 +356,17 @@ template <typename ToT, typename FromT> inline ToT cast_AS(FromT from) {
       return reinterpret_cast<ToT>(from);
 #endif // defined(__NVPTX__) || defined(__AMDGCN__)
   } else
+#ifdef __ENABLE_USM_ADDR_SPACE__
```

Comment:
Is this extension is only enabled if the user defines this macro?  That isn't consistent with other extensions and it isn't documented in the extension specification.

Reply:
My understanding is that if this isn't defined it will just use the same address space qualifier as for `global_space`, so the extension is still supported but does not have much of an effect.